tests/testutil/testutil.go

```diff
@@ -61,6 +61,11 @@ func ValidationPreCheck() {
 // Example: args=["run", "-n", "test", "nginx"] would run: "rio --namespace testing-namespace run -n test nginx"
 func RioCmd(args []string, envs ...string) (string, error) {
 	args = append([]string{"--namespace", TestingNamespace}, args...)
+	return RioExec(args, envs...)
+}
+
+// RioExec executes rio CLI commands with arguments, use RioCmd unless you have to use non testing ns
+func RioExec(args []string, envs ...string) (string, error) {
```

Comment:
Can we name this function something different? It's confusing since there is a `rio exec` command as well

Reply:
Yes, fixed.